.circleci/make-rpm-packages.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -eux
+
+echo "PARAM_RELEASE: $PARAM_RELEASE"
+echo "PARAM_DIST: $PARAM_DIST"
+
+if test "x$PARAM_RELEASE" = "x"; then
+    echo "Env variable PARAM_RELEASE is not set! For example PARAM_RELEASE=8, for CentOS 8"
+    exit 1
+elif test "x$PARAM_DIST" = "x"; then
+    echo "Env variable PARAM_DIST is not set! For example PARAM_DIST=centos"
+    exit 1
+fi
+
+if [ "$PARAM_DIST" = centos ]; then
+  if [ "$PARAM_RELEASE" = 8 ]; then
+      dnf install -y 'dnf-command(config-manager)'
+      yum config-manager --set-enabled PowerTools
+      yum install -y diffutils python3-sphinx
+  else
+      yum install -y python-sphinx
+  fi
+fi
+
+yum install -y epel-release
+yum install -y automake jemalloc-devel git libedit-devel libtool libunwind-devel make pcre-devel python3 sudo rpm-build yum-utils
+
+
+export DIST_DIR=build
+
+# XXX: we should NOT have to do that here, they should be in the
+# spec as BuildRequires
+yum install -y make gcc
+
+adduser varnish
+chown -R varnish:varnish /workspace
+
+cd /varnish-cache
+rm -rf $DIST_DIR
+mkdir $DIST_DIR
+
+
+echo "Untar redhat..."
+tar xavf /workspace/redhat.tar.gz -C $DIST_DIR
```

Comment:
I'd rather not have tight coupling and preexisting knowledge, can we just operate out of the current directory?

Reply:
I guess we can. The other scripts do not use `-C $DIST_DIR`. I just preserved the old code.
I will change it!

Reply:
I meant `/workspace/`. `DISTDIR` is a parameter that we could get rid of, but at least we can change it.

The fact that `/workspace/` is hardcoded means we can't run that script in parallel (outside of `circleci`). I think it makes more sense, if possible to go with

```
tar xavf redhat.tar.gz -C $DIST_DIR
```

Reply:
I've tried it with https://github.com/martin-g/varnish-cache/commit/1e1bb590ec64b4e5eaad57b9992cb860f0ab42b2 but it failed: https://app.circleci.com/pipelines/github/martin-g/varnish-cache/191/workflows/0b67272b-59ad-44df-9a4f-dd8d758e3b20/jobs/1039
It seems `rpmbuild` needs both a folder and a .tgz with the same name for `srcname`. 
It seems this is the reason redhat.tar.gz and varnish-cache.tar.gz to be extracted in `build/` folder.

Reply:
and that is fine, but I don't want to have to know that `redhat.tar.gz` is in `/workspace`. Same thing for the dist tarball.

```
tar xavf redhat.tar.gz -C $DIST_DIR
tar xavf varnish-*.tar.gz -C $DIST_DIR --strip 1
```

is perfectly fine, my beef with the current code is that it hardcodes the `/wokspace` when there's no need for it